pkg/quota/evaluator/core/pods.go

```diff
@@ -327,6 +377,21 @@ func isTerminating(pod *api.Pod) bool {
 	return false
 }
 
+func doesPriorityClassExist(pod *api.Pod) bool {
+	if pod.Spec.PriorityClassName != "" {
+		return true
+	}
+	return false
+}
+
+func isPriorityClassIn(pod *api.Pod, scope string) bool {
+	if pod.Spec.PriorityClassName != "" {
+		return false
+	}
+	//TODO: match after creating a set of priority classes from scope
+	return strings.Contains(scope, pod.Spec.PriorityClassName)
```

Comment:
This looks problematic to me. For example, if scope is "system-critical" and Pod priority class name is "critical", this function returns true, but IIUC, it should return false.